server/lib/activity-event.js

```diff
@@ -26,7 +26,6 @@ module.exports = function (event, data, request) {
 
   var eventData = _.assign({
     event: event,
-    path: removeQueryString(request.originalUrl),
```

Comment:
Nope. It was added during one of the earlier PRs for this feature, in response to [this comment](https://github.com/mozilla/fxa-content-server/pull/3587#discussion_r55832482). Back then, the events were emitted entirely server side and we did not know which page was responsible for instigating things. Now we do.

It isn't mentioned in [the spec](https://github.com/mozilla/fxa/tree/master/features/FxA-70-kpi-dashboards) and isn't needed to calculate the new KPIs.